requirements/dev.in

```diff
@@ -1,3 +1,4 @@
 -r base.in
 coverage
 pylint
+pytest
```

Comment:
Why did you add pytest?

Reply:
I had doubts, but decided to write tests using `pytest.`

Reply:
Marking as resolved.